src/main/java/duke/Event.java

```diff
@@ -0,0 +1,29 @@
+package duke;
```

Comment:
Like Deadline, you can consider putting this in a separate package within the Duke package so that it is more organised.